src/components/Profile/ProfileInfo/ProfileInfo.js

```diff
@@ -26,11 +26,11 @@ const ProfileInfo = (props) => {
 };
 
 ProfileInfo.defaultProps = {
-  profile: PropTypes.number,
+  profile: PropTypes.instanceOf(Object),
```

Comment:
See the comments above.

Reply:
I would prefer to do this upon completion of priority tasks